### DISCUSSION TITLE
Abandoned cart - Data are not capturing from ecom site to sfmc DE

### DISCUSSION THREAD

**Discussion by anuragkumar05** [SalesforceCommerceCloud/sfcc-ci#403] [Team Posts]
Hi,

We are trying to capture data from cart. if user added product in cart and left after some time data should be capture in MC data extension for that i have used below code but its not working.

<script type="text/javascript" src="https://MID.collect.igodigital.com/collect.js"> </script>
<script type="text/javascript">       
   _etmc.push(["setOrgId", "MID"]);       
   _etmc.push(["trackCart", { "cart": [
	{"item" : "INSERT_ITEM", "quantity":  "INSERT_QUANTITY" , "price" : "INSERT_PRICE" ,
            "unique_id" : "INSERT_UNIQUE_ID" } ,
	{"item" : "INSERT_ITEM", "quantity":  "INSERT_QUANTITY" , "price" : "INSERT_PRICE" ,
            "unique_id" : "INSERT_UNIQUE_ID" } 
   ]}]);   
</script>

please help how to get data from sfcc to mc data extension.

**Comment by zlatinz** [comment]
Abandon Cart functionality requires some non-trivial settings on the MC end. Please involve a MC Technical Architect from Salesforce to assist you with this.
